README.md

```diff
@@ -66,12 +66,30 @@ error (if it happened during an HTTP Request, of course)" (`RollbarRequest`),
 None of the fields on `RollbarBody` are updatable, and all null fields in
 `Rollbar.NET` are left off of the final JSON payload.
 
-## Example?
+## Examples
 
-This is a new library, hopefully more examples will be forthcoming.
+### Winforms
 
-Currently the best example of how to use this is at
-[Nancy.Rollbar](https://github.com/Valetude/Nancy.Rollbar). Please see the
-`RollbarPayloadFactory` and its dependencies.  To see how it might get
-integrated into a Nancy app you can take a peek in the unit tests for that
-project. There's a single example in the test suite, using Nancy's fakes.
+To use inside a Winforms Application, do the following inside your main method:
+
+```
+[STAThread]
+static void Main()
+{
+    Rollbar.Init(new RollbarConfig("POST_SERVER_ACCESS_TOKEN"));
```

Comment:
It defaults to production https://github.com/rollbar/Rollbar.NET/blob/improvements/Rollbar/RollbarConfig.cs#L12 but I'll add it to the example